conferences/abstracts/2021-03-CW21.md

```diff
@@ -0,0 +1,53 @@
+# Collaboration Workshop 2021
+
+* Website: https://www.software.ac.uk/cw21
+* Location: Online
+* Date: 30 March - 01 April 2021
+
+## Authors
+Malvika Sharan
+
+## Session collaborators
+TBA: SSI Fellows and volunteers from The Turing Way community
+
+## Mini-workshop or demo session title
+Good Practices for Designing Software Development Projects (The Turing Way)
+
+### Mini-workshop or demo session abstract
+Effective methods for project design are crucial to the success of a research project. 
+Particularly in software development, design principles can lead to better code, maintainability, and extendability. 
+Project design encompasses a variety of aspects, starting from defining the purpose, main research questions, expected users and target audience, available resources and skills required in the project. 
+It is also important to explore the possible outcomes, plans to address expected challenges, ensure diversity of stakeholders and reduce possible barriers to participation.
+
+The Turing Way is an open-source, community-led book project that aims to bring together diverse contributors and collaborators to share resources and practices that make data science reproducible, ethical and inclusive. 
+The project is developed and maintained on an online project repository (https://github.com/alan-turing-institute/the-turing-way) and invites contributions to its 5 guides, including the guide for project design.
```

Comment:
better to write 5 in words - five.

Also capitalise guide title in Guide for Project Design